### DISCUSSION TITLE
Winget MS Store automatic acceptance not working

### DISCUSSION THREAD

**Discussion by tankerkiller125** [microsoft/winget-cli#2750] [Q&A] [answered]
Some users are getting this prompt from winget:
![image](https://user-images.githubusercontent.com/3457368/205943263-6a0347d1-7bea-46a3-b203-d7eff8c5ad4c.png)

For additional context we're using the following command for installs:
`winget install --accept-package-agreements --accept-source-agreements --silent --exact "Microsoft.Office"`

This is on the first time Winget ever runs for these users (after freshly installing App Installer on Windows 10)

We can't seem to figure out how to fix this, and we don't see any options to add a command option or an environment variable to set the machine country code.

**Comment by jetrotal** [comment]
Hi, any news on that?

**Reply by denelon** [reply, marked as answer]
I believe this was fixed in a newer version of WinGet. It's likely that App Installer (the package WinGet is delivered in) need to be updated in the Microsoft Store, or installed via GitHub releases in this repository or via https://aka.ms/getwinget.
